### DISCUSSION TITLE
Welcome to nextcrm-app Discussions!

### DISCUSSION THREAD

**Discussion by pdovhomilja** [pdovhomilja/nextcrm-app#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ogc16** [comment]
👋 Hi everyone! Caleb here (@ogc16). I'm a software developer — I like to play around with code, and I plan on spending a good chunk of my time in open-source projects, making suggestions and implementing suggestions and feedback from the project.
My focus is building secure products in finance — currently implementing a bank API covering bookkeeping, financial planning, taxes, and revenue collections. I'm also running a fork of NextCRM , self-hosted and customized for my ecommerce setup, with the goal of feeding customers from Stripe purchases into the CRM and building a post-purchase email funnel (plus a second funnel for email-list signups). I'm still getting familiar with opensource.
Happy to share what I learn and contribute back where I can. Good to be here! 🤝
